docs/build.gradle

```diff
@@ -63,6 +63,7 @@ testClusters.matching { it.name == "integTest"}.configureEach {
   if (singleNode().testDistribution == DEFAULT) {
     setting 'xpack.license.self_generated.type', 'trial'
     setting 'indices.lifecycle.history_index_enabled', 'false'
+    systemProperty 'es.TIME_SERIES_MODE_FEATURE_FLAG_REGISTERED', 'true'
```

Comment:
I believe this is not required now that I've added https://github.com/elastic/elasticsearch/pull/87601/files#diff-dc8e1264edf1b5783bcf1aa7eeeaa4f4b4b5089fa97f951d1be97ccd604047b8R95 - which I merged this morning because I broke the build!

Reply:
Good catch. I've removed this line.